config-bootstrapper/src/main/resources/configs/config-bootstrapper/attribute-service/hypertrace/hypertrace_span_attributes.conf

```diff
@@ -279,6 +279,18 @@ commands = [
                 source_path: API_BOUNDARY_TYPE
               }
             },
+            {
+              fqn: Span.metrics.span_call,
+              key: spanCall,
+              value_kind: TYPE_INT64,
+              display_name: Span Call,
+              scope: EVENT,
+              sources: [QS],
+              onlyAggregationsAllowed: true,
+              supportedAggregations: [COUNT],
```

Comment:
AVGRATE should probably be allowed too (e.g. I want to query incoming spans/sec)

As far as the name, I don't have a great suggestion, but I don't love this since a span isn't really its own call. maybe just `Spans` so the query looks like `Count(spans)` ?

Reply:
`spans` make sense to me as well.

Reply:
`Count(spans)` makes sense but `spans` on its own isn't very intuitive ?
Maybe we can use spanTot, spanSum or something on similar lines ?

Reply:
"tot" and "sum" are each aggregations though, and the aggregation is declared independent of the attribute name. Because this is `onlyAggregationsAllowed: true`, it will never be used without an aggregation (although admittedly it's possible it may be selected first, before the aggregations are presented).

Reply:
So, how do we want to visualize here as an example below `Error Count`?
<img width="1051" alt="Screenshot 2021-11-11 at 11 13 49 AM" src="https://user-images.githubusercontent.com/53209990/141244312-35ead996-9453-4c1c-b57a-cb16bcd9f845.png">

So, should we have `Span Count` as display name or just `Spans`, what is making more sense? 

```
{
              fqn: Span.metrics.spans,
              key: spans,
              value_kind: TYPE_INT64,
              display_name: Span Count,
              scope: EVENT,
              sources: [QS],
              onlyAggregationsAllowed: true,
              supportedAggregations: [COUNT, AVGRATE],
              type: METRIC,
              internal: false
}
```

Reply:
I also think it makes sense to have different names for display and internal. Resolved using the above suggestions.

Reply:
Error Count is different - that's the number of errors in any particular span, right? Or is that binary and only an error or not? If so, I think I'd argue that It should still just be `Spans`, and `Error Count` should change to `Errors`

Reply:
Currently, Error Count is a binary - is span has an error (yes: 1 else :0 )?
- [Error Count Enricher link 1](https://github.com/hypertrace/hypertrace-ingester/blob/218e1ae07717ea5f9799ad117a21ef20aba82d30/hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/ErrorsAndExceptionsEnricher.java#L49), [Error Count Enricher link 2](https://github.com/hypertrace/hypertrace-ingester/blob/218e1ae07717ea5f9799ad117a21ef20aba82d30/hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/ErrorsAndExceptionsEnricher.java#L89)
- [spanEventViewGen link 1](https://github.com/hypertrace/hypertrace-ingester/blob/30f0a7b94e3bb01677df3a33fcf6e1c013515fc5/hypertrace-view-generator/hypertrace-view-generator/src/main/java/org/hypertrace/viewgenerator/generators/SpanEventViewGenerator.java#L29),   [spanEventViewGen link 2](https://github.com/hypertrace/hypertrace-ingester/blob/30f0a7b94e3bb01677df3a33fcf6e1c013515fc5/hypertrace-view-generator/hypertrace-view-generator/src/main/java/org/hypertrace/viewgenerator/generators/SpanEventViewGenerator.java#L224)

So, as your suggestion, let's go with `Spans` as a display name too.